addons/website_sale_stock/models/product_template.py

```diff
@@ -17,8 +17,8 @@ class ProductTemplate(models.Model):
     custom_message = fields.Text(string='Custom Message', default='')
 
     @api.multi
-    def _get_combination_info(self, combination=False, product_id=False, add_qty=1, pricelist=False, reference_product=False):
-        combination_info = super(ProductTemplate, self)._get_combination_info(combination, product_id, add_qty, pricelist, reference_product)
+    def _get_combination_info(self, combination=False, product_id=False, add_qty=1, pricelist=False, reference_product=False, only_template=False):
```

Comment:
Oops reference_product is actually named parent_combination in the parent method... Need the investigate the impact of that mistake and if it should be fixed.